cpp-client/README.md

```diff
@@ -41,6 +41,11 @@ C++ compiler and tool suite (cmake etc).
    Decide on a directory for the dependencies to live (eg, "$HOME/dhcpp").
    Create that directory and save the script there.
 
+   The two main build types of a standard cmake build are supported,
+   `Release` and `Debug`.  By default. `build-dependencies.sh`
+   creates a `Debug` build.  To create a `Release` build, set the
+   environment variable `BUILD_TYPE=Release` (1)
```

Comment:
I'm pretty sure this is `CMAKE_BUILD_TYPE` not `BUILD_TYPE`

Reply:
The variable build-depencies.sh reacts to, to in turn set `-DCMAKE_BUILD_TYPE=${BUILD_TYPE}`, is called `BUILD_TYPE`.
https://github.com/deephaven/deephaven-base-images/blob/7b5b909221232112c8e4b1b382e50c5df8d816dc/cpp-client/build-dependencies.sh#L370